### ISSUE TITLE
urls without host should be whiteListed by default

### ISSUE THREAD

**Issue by ahafsi** [auth0/angular2-jwt#518]
urls without host are not handled by the isWhitelistedDomain method. In fact it should return true when there is no host.

**Comment by Brenden-Morales** [comment]
Yeah this seems screwy. The documentation even says that this is a supported feature.

> For example, the following request assumes that the domain is the same as the one serving your app. It doesn't need to be whitelisted in this case.

`this.http.get('/api/things')
  .subscribe(...)`
